00_Base/src/services/commands.service.ts

```diff
@@ -49,22 +59,82 @@ export class CommandsService {
     }
   }
 
-  private handleCancelReservation(
-    _cancelReservation: CancelReservation,
-  ): OcpiResponse<CommandResponse> {
-    return ResponseGenerator.buildGenericClientErrorResponse({
-      result: CommandResponseType.NOT_SUPPORTED,
-      timeout: this.TIMEOUT,
-    });
+  private async handleCancelReservation(
+    cancelReservation: CancelReservation,
+    fromCountryCode: string,
+    fromPartyId: string,
+  ): Promise<OcpiResponse<CommandResponse>> {
+    try {
+      await this.commandExecutor.executeCancelReservation(
+        cancelReservation,
+        fromCountryCode,
+        fromPartyId,
+      );
+      return ResponseGenerator.buildGenericSuccessResponse({
+        result: CommandResponseType.ACCEPTED,
+        timeout: this.TIMEOUT,
+      });
+    } catch (e) {
+      if (e instanceof NotFoundError) {
+        return ResponseGenerator.buildGenericClientErrorResponse(
+          {
+            result: CommandResponseType.REJECTED,
+            timeout: this.TIMEOUT,
```

Comment:
This is a nitpick and me trying to understand the spec better. But what is the point of a 30s timeout here instead of 0? The rejection is pretty final and the timeout is about the result of the command.

Reply:
I remember @langiole helped share some info about this timeout before but I am cannot remember it clearly.

Reply:
I'm going to merge this because I don't think it's an issue either way, but would like to hear if there's a rationale @langiole